group-coordinator/src/main/java/org/apache/kafka/coordinator/group/GroupMetadataManager.java

```diff
@@ -932,6 +993,17 @@ private void throwIfConsumerGroupMemberEpochIsInvalid(
         }
     }
 
+    private void throwIfShareGroupMemberEpochIsInvalid(
```

Comment:
is this scenario possible in a share group heartbeat request that the group member has a member epoch greater than the one known by GC, hence we throw FencedMemberEpoch exception. Asking because I don't see any such FENCED_MEMBER_EPOCH exception in the share group heartbeat response API as per this [KIP-932 team review doc](https://confluentinc.atlassian.net/wiki/spaces/AK/pages/3292043446/KIP-932+team+review#Response-schema)